string-manipulation-1/src/get-last-char.js

```diff
@@ -0,0 +1,9 @@
+/* exported getLastChar */
+// storage for the last character in a string.
+// find and store the last character value of the string
+// give back the location of stored last character of said string
+function getLastChar(string) {
+  var lastChar = [];
```

Comment:
There's no need to make this value start as an empty array, you can just store the value you are retrieving on the line below in the variable as soon as you declare it.